app/routes.py

```diff
@@ -15,7 +15,8 @@
 from app.request import Request, TorError
 from app.utils.bangs import resolve_bang
 from app.utils.misc import read_config_bool, get_client_ip
-from app.utils.results import add_ip_card
+from app.utils.results import add_ip_card, check_currency
+from app.utils.results import add_currency_b
 from app.utils.results import bold_search_terms
```

Comment:
These could be condensed into a single import from `app.utils.results`. You'll probably have to break it up into multiple lines though (with `\`)

Reply:
oh okay. I seem to be having a lot of issues with pep8. Do you have any plugins or add-ons for code editors which I could use to make it easier?

Reply:
Which editor do you use? I use Vim, so I have [this plugin](https://github.com/tell-k/vim-autopep8) installed and configured to format on save. I also use Pycharm occasionally though, and that editor automatically shows pep8 formatting errors (although I don't think it enforces that you fix them). 

I'm sure there's other plugins if you don't use either of those editors, but I can't recommend any in particular. Alternatively you could set up a [pre-commit hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) to run the same pep8 check that the repo uses -- `pycodestyle app/*` (although that approach is a lot more work). 

Ultimately I wouldn't worry about it if you're getting errors from the pep8 part of the build. As long as it gets fixed before merge it's no big deal.

Reply:
I have been using Pycharm but it doesn't seem to care about the number of characters in the line. Like it allows me to type more than 72 characters. Maybe I need to set the limit myself? I also use sublime text editor. 

Well I have used sublime for the most part. Pycharm creates its .idea files and a virtual env so I find that annoying, deleting those before merging 😅.

 But then I am sure there is a way to avoid adding those files but I just don't have that much git knowledge I guess